_posts/2018-26-04-elixir-post.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title:  "Working with object oriented and functional programming without losing your mind."
+date:   2018-04-26 12:00:00 -0400
+categories: elixir coding ruby oo functional
+author: Tania Paiva
+---
+
+## Working with object oriented and functional programming without losing your mind.
```

Comment:
Titlecase. Also a tad long for the post's title, but what do I know.

Reply:
Length is fine for me. Drop the period at the end and capitalize, though.

Reply:
Another suggestion, split it into a subhead:
`Ruby + React + Elixir: 
Working with Object-oriented and Functional Programming Languages Without Losing Your Mind`